optimum/exporters/openvino/__main__.py

```diff
@@ -263,6 +263,44 @@ class StoreAttr(object):
         **loading_kwargs,
     )
 
+    if (
+        library_name == "diffusers"
+        and ov_config
+        and ov_config.quantization_config
+        and "dataset" in ov_config.quantization_config
```

Comment:
what if there is no `dataset` provided? can we allow such a behavior to enable weight quantization as well?

Reply:
> what if there is no dataset provided?

Weight quantization doesn't require the creation of an `OVxxx` pipeline.

> can we allow such a behavior to enable weight quantization as well?

It's already enabled.